packages/devtools_app/pubspec.yaml

```diff
@@ -0,0 +1,63 @@
+name: devtools_app
+description: Main package implementing web-based performance tooling for Dart and Flutter.
+
+# Note: when updating this version, please update the corresponding entry in
+# lib/devtools.dart and ../devtools/pubspec.yaml as well to the same version.
+# When publishing new versions of this package be sure to publish a new version
+# of package:devtools as well. package:devtools contains a compiled snapshot of
+# this package.
```

Comment:
maybe add a note about which package needs to be published first if order matters?

Reply:
doesn't matter as package_devtools doesn't depend on devtools_app